build.sh

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env bash
-docker build -t kartoza/postgis:manual-build .
-docker build -t kartoza/postgis:13.0 .
```

Comment:
Doesn't need to declare it. The dependencies are declared from the multi-stage build and declared in docker-compose file.